packages/vant/src/popup/Popup.tsx

```diff
@@ -145,6 +145,8 @@ export default defineComponent({
             duration={props.duration}
             customStyle={props.overlayStyle}
             onClick={onClickOverlay}
+            role={props.closeOnClickOverlay ? 'button' : undefined}
+            tabindex={props.closeOnClickOverlay ? 0 : undefined}
```

Comment:
Why not add role and tabindex to the Overlay component

Reply:
Because the overlay component is not necessarily used for user interaction, in some cases it may only be decoration. Therefore, if it is added to the overlay, some situations will send some information to the visually impaired user incorrectly.